src/resources/2024/BatWings.ts

```diff
@@ -38,3 +39,19 @@ export function cauldronsRemaining(): number {
 export function flapChance(flaps = get("_batWingsFreeFights")): number {
   return flaps < 5 ? 1 / (2 + flaps) : 0;
 }
+
+/**
+ * Attempts to do bridge skip
+ * @returns whether we were successfully able to skip the bridge
+ */
+export function bridgeSkip(): boolean {
+  if (get("chasmBridgeProgress") < 25 || get("questL09Topping") !== "started") {
+    return false;
+  }
+  visitUrl(
+    `place.php?whichplace=orc_chasm&action=bridge${get("chasmBridgeProgress")}`,
```

Comment:
Trying to build the bridge here doesn't really make sense given that the first clause of the if statement blocks getting here if they haven't built enough already. We should decide whether we want to require callers to handle their own bridge progress or if libram is responsible for handling that for them and be consistent.

(I'm in support of libram helping build the bridge too, but that's more up to @horrible-little-slime.)